File: java/learning-spring-01/src/main/java/com/example/learningspring/beans/game/items/SuperContraGame.java
package com.example.learningspring.beans.game.items;

public class SuperContraGame implements GameConsole {
    public void up() {
        System.out.println("Jump");
    }

    public void down() {
        System.out.println("Go into a hole");
    }

    public void left() {
        System.out.println("Go left");
    }

    public void right() {
        System.out.println("Accelerate");
    }
}

File: kotlin/learning-spring/src/main/kotlin/com/examplekotlin/learningspring/coupled/loosely/game/SuperContraGame.kt
package com.examplekotlin.learningspring.coupled.loosely.game

class SuperContraGame : GameConsole {
    override fun up() {
        println("Jump")
    }

    override fun down() {
        println("Go into a hole")
    }

    override fun left() {
        println("Go left")
    }

    override fun right() {
        println("Accelerate")
    }
}